src/OrchardCore.Modules/Orchard.Autoroute/Handlers/AutoroutePartHandler.cs

```diff
@@ -119,5 +124,29 @@ private string GetPattern(AutoroutePart part)
 
             return pattern;
         }
+
+        private string GenerateUniquePath(string path, AutoroutePart context)
```

Comment:
You might think it's simple, until you realize it's even simpler to just do nothing, which will prevent users from finding issues with your code, and you from having to fix them ;)